.prettierrc

```
@@ -0,0 +1,6 @@
+{
+  "bracketSpacing": true,
+  "singleQuote": true,
+  "tabWidth": 2,
+  "arrowParens": "avoid"
```

```suggestion
  "arrowParens": "avoid",
  "useTabs": false
```
This one is important, learned from @PixnBits a long time ago that spacing is the way to go for git history reasons.

---

As I said, it's up to everyone what you want the defaults to be. I'm not a fan of brace spacing, but I'm willing to change if you guys are willing to enforce rules and not allow opting out (part of a larger discussion).

---

I think we do want a baseline opinion on how prettier should be configured. Extending and overriding can be done via `.prettierrc` in the repository it is used in.